### PR TITLE
Add the gsoc blog post

### PR DESCRIPTION
**Pull request by peey**
From medium: https://medium.com/@peey/personal-experiences-at-babel-1-a-pr-with-unusually-high-number-of-reviews-5cb49ee71897

cc @hzoo

### CODE REVIEW

File: _posts/2017-07-02-gsoc-peey-1.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+Well, there is more to the story. As I was recently discussing with a mentor,
+reviewing PRs is a tough job. Reviewing PRs is comparably as hard as solving the
+bug in the first place was. Apart from the technical aspect of reviewing — which
+is ensuring that the bug is being fixed optimally (by perhaps even getting an
+idea of how they would solve the issue and seeing how the submitted patch
+compares to their idea) — there’s another big hindrance. A reviewer has to be
+aware of the whole issue, the discussion surrounding it, and have familiarity
+with the part of the codebase that the PR makes changes to.<sup><a href="#footnotes">1</a></sup>
+
+When I initially joined Babel, and was not-so-familiar with the codebase, every
```
Comment (existentialism) [inline _posts/2017-07-02-gsoc-peey-1.md]: > and that everyone’s expected to adhere to _the_ same fictitious standards

File: _posts/2017-07-02-gsoc-peey-1.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+able to evaluate it just like they were answering my questions.
+
+Eh! Very wrong. Let me just bust this myth (assuming I’m not the only one who
+has felt it). Even they (maintainers) won’t have all the answers at times, and
+sometimes you’ll have to search for yourself — and that’s how it should be.<sup><a href="#footnotes">2</a></sup>
+
+In open source, there are a lot of people who want to contribute, but are unable
+to because they don’t know how to code / they don’t know how to present PRs /
+they don’t know what the project wants / they don’t know what the maintainers
+want / a ton of other things. A lot of the times you’ll find help along the way,
```
Comment (existentialism) [inline _posts/2017-07-02-gsoc-peey-1.md]: > A lot of ~~the~~ times you’ll find help along the way

File: _posts/2017-07-02-gsoc-peey-1.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+but a lot of that is controlled by factors beyond your control.<sup><a href="#footnotes">3</a></sup>
+
+One of the joys of getting your PR merged is not just the programming but
+somehow making the project move forward in the way it is expected to be moved
+forward. And doing that by somehow identifying what the project needs and being
+able to deliver that.
+
+In order to merge this PR I had find people and talk to them — people who use
```
Comment (existentialism) [inline _posts/2017-07-02-gsoc-peey-1.md]: > In order to merge this PR I had _to_ find people

also, babel -> Babel

> spec and all the existing _discussions_ surrounding

File: _posts/2017-07-02-gsoc-peey-1.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+2.  The discussion surrounding it (by mentioning alternate viewpoints on some of the
+decisions so as to make it easier to compare them with the decisions made)
+3.  Explaining clearly my strategy to solve the problem (to assist the technical
+aspect of reviewing — so that the reviewers can know what I’ve done and then see
+the code rather than the other way around)
+
+And that’s what did it! (or so I think). So there’s the mystery unraveled — A PR
+with unusually high number of reviews¹.
+
+**PS**: I wanted to share my personal experience experience with this blog post,
```
Comment (existentialism) [inline _posts/2017-07-02-gsoc-peey-1.md]: Nit: extra experience

File: _posts/2017-07-02-gsoc-peey-1.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+not write a guide to be followed or a technical blog post. Therefore, some
+statements which I make may not hold true in general or may be debatable, so
+they should just be read in the context of the experience I narrate.
+
+Also note that if you’re looking for decorators support in Babel, we still have
+a long way to go. This is just the parser and work on the transform (which
+converts your code to functionally equivalent ES5)is yet to be done. But now
```
Comment (existentialism) [inline _posts/2017-07-02-gsoc-peey-1.md]: Missing space between `ES5)is`

File: _posts/2017-07-02-gsoc-peey-1.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+smoothly from here onwards.
+
+## Footnotes
+
+1.  We have a shortage of manpower when it comes to reviewing PRs. It was also
+recently discussed in one of our weekly meetings ([link to meeting
+notes](https://github.com/babel/notes/blob/master/2017-06/june-21.md#priority-topics)).
+Perhaps you could help us with this. Drop by [our slack
+chatroom](https://slack.babeljs.io/) and offer your help!
+2.  I feel that the myth stems from the fact that when you’re new to the project the
```
Comment (existentialism) [inline _posts/2017-07-02-gsoc-peey-1.md]: `so` at the end seems extraneous?

**Comment by hzoo** [comment]
we should mention somewhere that you are part of gsoc (you can ignore the ci fail, looks like a lot of random remark errors). (I should just write up a blogpost about how we are participating in that actually)

**Comment by peey** [comment]
@existentialism fixed those! @hzoo : added that info.

Also decided to run it through Grammarly and found that I've been using too many commas which aren't needed. I hope they can stay, because when I wrote this, I was using them as conversation pauses!

**Comment by Daniel15** [comment]
This is failing lint, which is breaking the Travis build. Could you please fix it?

```
- ./_site/blog/2017/08/11/personal-experiences-at-babel-1-a-pr-with-unusually-high-number-of-reviews.html
  *  internally linking to #footnotes, which does not exist (line 9)
     <a href="#footnotes">1</a>
  *  internally linking to #footnotes, which does not exist (line 9)
     <a href="#footnotes">2</a>
  *  internally linking to #footnotes, which does not exist (line 9)
     <a href="#footnotes">3</a>
  *  internally linking to #footnotes, which does not exist (line 9)
     <a href="#footnotes">4</a>
rake aborted!
HTML-Proofer found 4 failures!
```

**Pull request merged**